### PR TITLE
Placeholder blurry disable

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@andreban what do you think of this? It's a breaking change, but I think we should make it.

File: packages/optimizer/README.md [hunk 1]
```diff
@@ -83,7 +83,8 @@ You can find a sample implementation [here](demo/simple/). If you're using the e
 Currently the following options are supported:
 
 * **ampUrl**: an URL string pointing to the valid AMP version. Required by the AddAmpLink transformer.
-* **imageBasePath**: a base URL or path required for resolving an image file from a given image `src` attribute. Required by the AddBlurryImagePlaceholder transformer.
+* **blurredPlaceholders**: Enables blurry image placeholder generation. Default is `false`. Make sure to only enable  **Important:** blurry image placeholder computation is expensive. Make sure to only use it for static or cached pages.
```
Comment (andreban) [inline packages/optimizer/README.md]: This sentence looks incomplete:  "make sure to only enable"

Comment (andreban) [reply packages/optimizer/README.md]: Also, for consistency: s/Enable/enable

**Comment by andreban** [comment]
+1. I think folks using the blurredPlaceholder should make a conscious decision.

File: packages/optimizer/README.md (hunk 1 above)
Comment (sebastianbenz) [reply packages/optimizer/README.md]: done

**Comment by sebastianbenz** [comment]
@andreban PTAL

**Pull request merged**